JASP-Engine/JASP/R/bainttestbayesianindependentsamples.R

```diff
@@ -122,103 +121,104 @@ BainTTestBayesianIndependentSamples <- function(jaspResults, dataset, options, .
+                               "BF[less]" = BF_01,
+                               "pmp[less]" = PMP_1)
         }
-    } else if (options$bayesFactorType == "BF10"){
-        if(options$hypothesis == "groupsNotEqual"){
-            row <- list(Variable=variable, "hypothesis[type1]" = "H0: Equal","BF[type1]"="", "pmp[type1]" = .clean(PMP_0),
-                                "hypothesis[type2]" = "H1: Not equal", "BF[type2]" = .clean(BF_0u), "pmp[type2]" = PMP_u)
-        } else if(options$hypothesis == "groupOneGreater"){
-            row <-list(Variable=variable, "hypothesis[type1]" = "H0: Equal","BF[type1]"="", "pmp[type1]" = .clean(PMP_0),
-                               "hypothesis[type2]" = "H1: Bigger", "BF[type2]" = .clean(BF_01), "pmp[type2]" = .clean(PMP_1))
-        } else if(options$hypothesis == "groupTwoGreater"){
-            row <-list(Variable=variable, "hypothesis[type1]" = "H0: Equal", "BF[type1]"="", "pmp[type1]" = .clean(PMP_0),
-                               "hypothesis[type2]" = "H1: Smaller", "BF[type2]" = .clean(BF_01), "pmp[type2]" = .clean(PMP_1))
-        } else if (options$hypothesis == "_4type"){
-            row <-list(Variable=variable, "hypothesis[type1]" = "H1: Bigger", "BF[type1]"= "", "pmp[type1]" = .clean(PMP_1),
-                               "hypothesis[type2]" = "H2: Smaller", "BF[type2]" = .clean(BF_01), "pmp[type2]" = .clean(PMP_0))
-        } else if (options$hypothesis == "allTypes"){
+    } else if (options$bayesFactorType == "BF10") {
+        if (options$hypothesis == "groupsNotEqual") {
+            row <- list(Variable=variable, "hypothesis[type1]" = "H0: Equal","BF[type1]"="", "pmp[type1]" = PMP_0,
+                                "hypothesis[type2]" = "H1: Not equal", "BF[type2]" = BF_0u, "pmp[type2]" = PMP_u)
+        } else if (options$hypothesis == "groupOneGreater") {
+            row <-list(Variable=variable, "hypothesis[type1]" = "H0: Equal","BF[type1]"="", "pmp[type1]" = PMP_0,
+                               "hypothesis[type2]" = "H1: Bigger", "BF[type2]" = BF_01, "pmp[type2]" = PMP_1)
+        } else if (options$hypothesis == "groupTwoGreater") {
+            row <-list(Variable=variable, "hypothesis[type1]" = "H0: Equal", "BF[type1]"="", "pmp[type1]" = PMP_0,
+                               "hypothesis[type2]" = "H1: Smaller", "BF[type2]" = BF_01, "pmp[type2]" = PMP_1)
+        } else if (options$hypothesis == "_4type") {
+            row <-list(Variable=variable, "hypothesis[type1]" = "H1: Bigger", "BF[type1]"= "", "pmp[type1]" = PMP_1,
+                               "hypothesis[type2]" = "H2: Smaller", "BF[type2]" = BF_01, "pmp[type2]" = PMP_0)
+        } else if (options$hypothesis == "allTypes") {
             row <-list(Variable=variable,
                                "type[equal]" = "H0: Equal",
                                "BF[equal]"= "",
-                               "pmp[equal]" = .clean(PMP_0),
+                               "pmp[equal]" = PMP_0,
```

Comment:
I guess that cleaning is not necessary anymore, because that's done by jaspResults. Is that correct?

Reply:
Yep jaspResults does this now in the aptly named macro `TO_INFINITY_AND_BEYOND`